### PR TITLE
Enable and disable ha proxy service using systemctl commands

### PR DESCRIPTION
**Pull request by manasagowri**
Signed-off-by: mgowri <mgowri@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/pacific/cephadm/test-haproxy-deployment.yaml [hunk 1]
```diff
@@ -187,3 +187,76 @@ tests:
...
+      config:
+        sudo: True
+        check_status: True
+        role: haproxy
+        service_name: haproxy
+        commands:
+          - "systemctl enable"
+  - test:
+      name: check-ceph-health and crash status on client node5
+      desc: Check for ceph health and crash status after ha proxy has be reenabled
```
Comment (sunilangadi2) [inline suites/pacific/cephadm/test-haproxy-deployment.yaml:254]: **has been** there is spelling mistake Please correct that?

File: suites/pacific/cephadm/test-container-cli-args.yaml [hunk 2]
```diff
@@ -267,7 +267,7 @@ tests:
       config:
         command: add
         id: client.1                      # client Id (<type>.<Id>)
-        node: node4                       # client node
+        node: node5                       # client node
```
Comment (sunilangadi2) [inline suites/pacific/cephadm/test-container-cli-args.yaml:270]: if you are planning to change the node name please update the node name in test desc as well.

**Review by sunilangadi2 — CHANGES_REQUESTED** [review]
@manasagowri Can you please add test run results?

**Comment by manasagowri** [comment]
> @manasagowri Can you please add test run results? Also add the **polarion id** to the respective test case you added.

Success logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-LPOL8C/

Polarion IDs added

File: suites/pacific/cephadm/test-haproxy-deployment.yaml (hunk 1 above)
Comment (manasagowri) [reply suites/pacific/cephadm/test-haproxy-deployment.yaml]: Done.

File: suites/pacific/cephadm/test-container-cli-args.yaml (hunk 2 above)
Comment (manasagowri) [reply suites/pacific/cephadm/test-container-cli-args.yaml:271]: Done.

**Review by sunilangadi2 — APPROVED** [review]
Changes LGTM.

File: tests/misc_env/exec.py [hunk 3]
```diff
@@ -134,6 +134,28 @@ def translate_to_daemon_id(node, string: str) -> str:
...
+        command after adding the service name.
+    """
+    cmd = "touch grepout"
+    out, _ = exec_command(node, sudo=True, cmd=cmd)
+    cmd = "systemctl -l | grep ceph > /root/grepout"
+    out, _ = exec_command(node, sudo=True, cmd=cmd)
+    cmd = f"cat /root/grepout | grep {service_name}"
+    out, err = exec_command(node, sudo=True, cmd=cmd)
+    service_full_name = out.split()[0]
+    return f"{command} {service_full_name}"
```
Comment (psathyan) [inline tests/misc_env/exec.py:156]: I believe we can do this with a single call
```
# systemctl list-units --type=service | grep ceph | grep alertmanager | cut -d " " -f 1

cmd = f"systemctl list-units --type=service | grep ceph | grep {service_name} | cut -d ' ' -f 1"
out, err = exec_command(node, cmd)
return out.split()[0]
```

Comment (manasagowri) [reply tests/misc_env/exec.py]: Changes done - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4GE5AP/

**Review by psathyan — APPROVED** [review]

**Pull request merged**